core/domain/translation_services_test.py

```diff
@@ -289,6 +296,139 @@ def test_compute_translation_related_change_mark_translation_needs_update(
             ], [False, True]
         )
 
+    def test_compute_translation_related_change_edits_existing_translation(
+        self
+    ) -> None:
+        translation_services.add_new_translation(
+            feconf.TranslatableEntityType.EXPLORATION, self.EXP_ID, 5, 'hi',
+            'content_5', translation_domain.TranslatedContent(
+                'Translations in Hindi!',
+                translation_domain.TranslatableContentFormat.HTML,
+                False
+            )
+        )
+        translation_services.add_new_translation(
+            feconf.TranslatableEntityType.EXPLORATION, self.EXP_ID, 5, 'hi',
+            'content_6', translation_domain.TranslatedContent(
+                'Translations in Hindi!',
+                translation_domain.TranslatableContentFormat.HTML,
+                False
+            )
+        )
+
+        entity_translation_models: Sequence[
+            translation_models.EntityTranslationsModel
+        ] = translation_models.EntityTranslationsModel.get_all().fetch()
+        self.assertEqual(len(entity_translation_models), 1)
+        entity_translation_model = entity_translation_models[0]
+        self.assertEqual(entity_translation_model.entity_version, 5)
+        self.assertEqual([
+            t['needs_update']
+            for t in entity_translation_model.translations.values()
+        ], [False, False])
```

Comment:
Nit: remove the new lines above and below to consolidate the test setup phase or add comments to describe each "section". Ditto below.

Reply:
This is not setup it's the baseline verification. This test has the following sections separated by newlines: 1. Base setup 2. Basline verification 3. Setup for test 4. Action 5. Final verififcation

Reply:
I've added a new line between "4. Action 5. Final verification"